docs/release_notes/v0.16.0.rst

```diff
@@ -0,0 +1,30 @@
+SMQTK-Classifier Pending Release Notes
+======================================
```

Comment:
Rename title
```suggestion
v0.16.0
=======
```

Reply:
ah right I'll have to fix this in the image io release as well